pkg/apiserver/experiment/experiment.go

```diff
@@ -144,8 +135,64 @@ func (s *Service) listExperiments(c *gin.Context) {
 // TODO: need to be implemented
 func (s *Service) getExperimentDetail(c *gin.Context) {}
 
-// TODO: need to be implemented
-func (s *Service) deleteExperiment(c *gin.Context) {}
+// @Summary Delete the specified chaos experiment.
+// @Description Delete the specified chaos experiment.
+// @Tags experiments
+// @Produce json
+// @Param namespace path string true "namespace"
+// @Param name path string true "name"
+// @Param kind query string true "kind" Enums(PodChaos, IoChaos, NetworkChaos, TimeChaos, KernelChaos, StressChaos)
+// @Success 200 "delete ok"
+// @Router /api/experiments/{namespace}/{name} [delete]
+// @Failure 400 {object} utils.APIError
+// @Failure 500 {object} utils.APIError
+func (s *Service) deleteExperiment(c *gin.Context) {
+	kind := c.Query("kind")
```

Comment:
Why not add this to `Param' ?

Reply:
I am not sure about the pattern. The URL might be too long.

Reply:
They are the same variables, so I think we need to unify these.

Reply:
Do you prefer `/:ns/:name/:kind` or `/:kind/:ns/:name`?

Reply:
`/:kind/:ns/:name`?